### DISCUSSION TITLE
BOLD5000 fMRI dataset

### DISCUSSION THREAD

**Discussion by crcox** [pykale/pykale#296] [Student projects]
The BOLD5000 dataset contains fMRI data for four subjects who were each presented ~5000 images that were taken from the corpus used to train ImageNet. The images are quite naturalistic and diverse, which differentiates that data from other sets on might find on OpenNeuro.

A summary introduction to the dataset can be found here: https://bold5000-dataset.github.io/website/index.html
Source code for the processing of the data can be found here: https://github.com/BOLD5000-dataset/BOLD5000
The publication promoting the dataset can be found here: https://www.nature.com/articles/s41597-019-0052-3
The latest version of the dataset itself can be found here: https://figshare.com/articles/dataset/BOLD5000_Release_2_0/14456124

The major aim of collecting and sharing this large dataset was to compare distributed representations responsible for encoding these images in ImageNet (the computational model) and the brain. Highlighting points of similarity and difference between real and artificial neural networks can advance our understanding of the brain represents images and associated meaning (sometimes called "semantic content").

The dataset provides many opportunities for secondary analysis. Because there are multiple participants and a relevant pre-trained computational model, there may be interesting projects that can be designed around transfer learning/domain adaptation.

**Comment by shuo-zhou** [comment]
Hello Chris, I have read the BOLD5000 Nature paper. The followings are my ideas of possible student projects:

- Brain decoding (e.g. scene categories) within participants using machine learning.
- Domain adaptation / generalisation for brain decoding across participants (CSI1-CSI4). For example, train a decoding model with the data of CSI1, CSI2, and CSI3, and then test on CSI4.

Please let me know if you have any suggestion / idea.

**Comment by haipinglu** [comment]
Reference:
1. EMBC21 https://ieeexplore.ieee.org/abstract/document/9630360/
2. https://repository.ubn.ru.nl/bitstream/handle/2066/228249/228249.pdf?sequence=1 
3. https://www.proquest.com/docview/2542456809?pq-origsite=gscholar&fromopenview=true
4. https://www.proquest.com/docview/2393755010?pq-origsite=gscholar&fromopenview=true
